ankiconnector.cpp

```diff
@@ -50,6 +50,23 @@ void AnkiConnector::sendToAnki( QString const & word, QString const & text, QStr
                                        Utils::json2String( fields ) );
 
 //  qDebug().noquote() << postData;
+    postToAnki( postData );
+}
+
+void AnkiConnector::ankiSearch( QString const & word )
+{
+    QString postTemplate = R"anki({
```

Comment:
if anki connect is not enabled ,can return directly?

Reply:
> what will happen,if no anki installed and click on the ankisearch link

If Ankiconnect isn't running or if Anki isn't runnning, it prints "connection refused" at the bottom.

Reply:
> if anki connect is not enabled ,can return directly?

Cancel the operation if ankiconnect isn't enabled in settings? i'll look into it.

Reply:
Added an `if` that returns if ankiconnect isn't enabled in settings.